hadoop-ozone/dist/src/main/compose/ozonesecure-ha/docker-config-om-ratis

```diff
@@ -0,0 +1,20 @@
+# Licensed to the Apache Software Foundation (ASF) under one
+# or more contributor license agreements.  See the NOTICE file
+# distributed with this work for additional information
+# regarding copyright ownership.  The ASF licenses this file
+# to you under the Apache License, Version 2.0 (the
+# "License"); you may not use this file except in compliance
+# with the License.  You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+OZONE-SITE.XML_ozone.om.ratis.log.purge.gap=50
```

Comment:
How about `docker-config-ratis-om-bootstrap`? These are generic config for ratis to expedite the bootstrap Ratis snapshot process. It's not an override because these configs are appended to the other configs.

Reply:
`docker-config-ratis-om-bootstrap` is reasonable.

Reply:
Done.